### PR TITLE
MBS-9470: Expose UUID of relationship attribute type

### PR DESCRIPTION
**Pull request by reosarevok**
Goes with https://github.com/metabrainz/mmd-schema/pull/16

For JSON, created new attribute-ids section. This is not ideal - we *already* have two "number" attributes with different meanings and IDs which will be indistinguishable from one another when using name as key - but it's probably the only non-breaking option.

### CODE REVIEW

**Comment by mwiencek** [comment]
Merged the mmd-schema changes. Can you look into the test failures? :)

**Comment by reosarevok** [comment]
This seems to be passing tests locally... so I'm not sure why it's failing here.

**Comment by mwiencek** [comment]
Updating `MMD_SCHEMA_COMMIT` seems to fix things.

File: lib/MusicBrainz/Server/WebService/XMLSerializer.pm [hunk 1]
```diff
@@ -1061,11 +1061,11 @@ sub _serialize_relation
     push @list, $gen->attribute_list(
         map {
             if (non_empty($_->text_value)) {
-                $gen->attribute({ value => $_->text_value }, $_->type->name);
+                $gen->attribute({ value => $_->text_value, id => $_->type->gid }, $_->type->name);
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/WebService/XMLSerializer.pm]: I think `type-id` is more consistent with the rest of our XML than `id`.

**Comment by mwiencek** [comment]
Re-fixed mmd-schema, LGTM now.

**Pull request merged**